palindrom-client.html

```diff
@@ -232,12 +232,7 @@
                     console.warn(`palindrom-client has noticed a huge patch of ${patchesAndResults.patches.length} operations. For better performance palindrom-client will replace the whole state object instead of applying your updates individually`);
```

Comment:
I say remove this whole if. We agreed that it's too hacky and didn't help Palma.

Reply:
oh, really? Could we make it a separate PR?

Reply:
Separated issue to https://github.com/Palindrom/palindrom-client/issues/80